ugvc/mrd/srsnv_plotting_utils.py

```diff
@@ -84,24 +103,48 @@ def create_data_for_report(
     return df, df_tp, df_fp, max_score, cls_features, fprs, recalls
 
 
-def srsnv_training_report(
+def srsnv_report(
     out_path,
     out_basename,
     report_name,
-    model_path,
-    X_path,
-    y_path,
-    fp_bed_file,
-    params_path,
+    model_file,
+    params_file,
     simple_pipeline=None,
 ):
     reportfile = pjoin(out_path, f"{out_basename}{report_name}_report.ipynb")
     reporthtml = pjoin(out_path, f"{out_basename}{report_name}_report.html")
 
+    [
+        output_roc_plot,
+        output_LoD_plot,
+        output_cm_plot,
+        output_precision_recall_qual,
+        output_qual_density,
+        output_obsereved_qual_plot,
+        output_ML_qual_hist,
+        output_qual_per_feature,
+        output_bepcr_hists,
+        output_bepcr_fpr,
+        output_bepcr_recalls,
+    ] = _get_plot_paths(report_name, out_path=out_path, out_basename=out_basename)
+
     commands = [
-        f"papermill ugvc/reports/srsnv_training_report.ipynb {reportfile} -p report_name {report_name} \
-            -p model_file {model_path} -p X_file {X_path} -p y_file {y_path} \
-            -p single_sub_regions {fp_bed_file} -p params_file {params_path} -k python3",
+        f"papermill ugvc/reports/srsnv_report.ipynb {reportfile} \
+        -p {report_name} \
+        -p {model_file} \
+        -p {params_file} \
+        -p {output_roc_plot} \
+        -p {output_LoD_plot} \
+        -p {output_cm_plot} \
+        -p {output_precision_recall_qual} \
+        -p {output_qual_density} \
+        -p {output_obsereved_qual_plot} \
+        -p {output_ML_qual_hist} \
+        -p {output_qual_per_feature} \
+        -p {output_bepcr_hists} \
+        -p {output_bepcr_fpr} \
+        -p {output_bepcr_recalls} \
+        -k python3",
         f"jupyter nbconvert {reportfile} --output {reporthtml} --to html --template classic --no-input",
```

Comment:
The "--template classic" argument is deprecated, you can remove it

Reply:
done